syz-cluster/pkg/api/api.go

```diff
@@ -22,6 +22,7 @@ type FuzzTask struct {
 
 // FuzzConfig represents a set of parameters passed to the fuzz step.
 type FuzzConfig struct {
+	Suffix    string `json:"suffix"` // E.g. KASAN.
```

Comment:
Suffix is how you use it.
I think something like "type" or "test_type" will improve the readability here.